docs/cve-creation-guide.md

```diff
@@ -0,0 +1,189 @@
+
+## Research the CVE database to ensure it has not been logged previously
+
+Ensuring that the issue has not been previously logged is very important.
+Duplicate entries will merely serve to confuse the issue and polute the CVE database.
+
+1. Access [Mitre.org CVE Search](https://cve.mitre.org/cve/search_cve_list.html)
+1. Search for various keywords including but not limited to the distribution or module name and any applicable words.
+Review the results
+1. If the CVE appears to be the correct one it would likely be best to submit an issue containing the CVE number and link to the CVE database to  [https://github.com/briandfoy/cpan-security-advisory/issues](https://github.com/briandfoy/cpan-security-advisory/issues)
+
+## Access Mitre CVE Form
+
+1. Access [Mitre CVE Form](https://cveform.mitre.org/)
+1. Select a request type: **_Report Vulnerability/Request CVE ID_**
+1. Enter your e-mail address: **_cpan-security@perl.org_**
+1. Enter a PGP Key (to encrypt): **_Leave blank as the email address is a mailing list_**
+1. Number of vulnerabilities reported or IDs requested (1-10): **_1_**
+
+### Vulnerability Type
+This field may take some research/discussion with the CPAN module maintainer.
+In the case of CPANSA-Mojolicious-2018-01 it mentions *leaks old cookies* so it is fairly obvious that it is _Information Disclosure_ however that is not on the list.
+Options here that look like potentials are:
+
+1. Insecure Permissions
+1. Incorrect Access Control
+1. Other or Unknown
+
+According to the [Pull Request](https://github.com/mojolicious/mojo/pull/1192):
+
+_Mojolicious does not set the cookie domain if it is unset, but instead stores an additional origin parameter to store the original request-host.
+RFC 6265 also requires that cookies without a valid domain set are assigned the domain of their request-host_
+
+Based on the lack of a **domain** attribute I would tend to specify **_Insecure Permissions_**
+
+**_Insecure Permission_** is defined as _"The software, upon installation, sets incorrect permissions for an object that exposes it to an unintended actor."_
+
+In this case the lack of the **domain** attribute exposes the old cookies to the server.
+
+This is a judgement call so either of the other choices might be okay.
+
+1. Vulnerability type: **_Insecure Permissions_**
+
+
+### Vendor of the product(s)
+
+Here you should specify the maintainer of the module.
+Although you may be requesting CVE IDs for older vulnerabilities  you can specify the current maintainer as it does not seem to propogate o the CVE database.
+
+1. Vendor of the product(s): **_SRI on CPAN_**
+
+### Product
+
+For some CPAN modules this is fairly simple.
+For others you may need to do some additional research.
+For _CPANSA-Mojolicious-2018-01_ CPAN::Audit lists the vulnerability as **Mojolicious** in the identifier but the description correctly identifies **_Mojo::UserAgent::CookieJar_**.
```

Comment:
I'd say that in this example `Mojolicious` is the distribution name, and the product that contains the vuln.

Reply:
Yes, but in the case of a module like this it is specific to a certain part so it may be a case of a vulnerabilit that you are not encountering if you are not using Mojo::UserAgent::CookieJar